Kernel/Process.cpp

```diff
@@ -239,6 +239,25 @@ static bool validate_mmap_prot(int prot, bool map_stack)
     return true;
 }
 
+// Split a region into two where one of the Regions contains the entire Range
```

Comment:
This comment seems confusing and wrong.
What we're really doing is splitting a `Region` *around* a `Range`, returning the one or two remaining `Region`s.

Perhaps a better name would be `split_region_around_range`?

Reply:
Right, I completely misinterpreted this code before. Renamed to split_region_around_range and fixed the comment.